server/datastore/mysql/scheduled_queries.go

```diff
@@ -145,3 +164,35 @@ func (d *Datastore) ScheduledQuery(id uint) (*fleet.ScheduledQuery, error) {
 
 	return sq, nil
 }
+
+func (d *Datastore) ScheduledQueryByQueryAndPack(queryID uint, packID uint) (*fleet.ScheduledQuery, error) {
```

Comment:
Is this new datastore method needed? Can global schedule queries be modified by their `scheduled_queries` ID like other scheduled queries?

Reply:
My understanding is that the idea would be:

- You create a Query
- You add it to the global Schedule

No ScheduledQuery or Pack needed, that's the complexity we hide here.